generators/bitrise/README.md

```diff
@@ -2,7 +2,15 @@
 
 :warning: ***Having set up Fastlane with `yo rn-toolbox:fastlane` is a requirement***
 
-## 1/ Create a new app on Bitrise
+## 3 files?
+
+After testing a single build for both platforms (`bitrise/bitrise.common.yml`) we were still disappointed with the deployment time. In order to obtain a faster build we decided on splitting the builds between iOS and Android.
```

Comment:
Est-ce qu'on ne peut pas plutôt laisser le choix aux utilisateurs dans le générateur, qqch du genre: 
Do you want to generate separate build configs for Android & iOS?